style.css

```diff
@@ -41,9 +65,10 @@ div.navbar {
 }*/
 
 footer {
-   margin-top: 50px;
+   margin-top: 550px;
```

Comment:
Though I've seen it in other projects, one of those things I haven't yet been properly introduced to and subsequently never used is rem. Thanks for the tip.